### DISCUSSION TITLE
Student Developer Pack Approved But Access Denied

### DISCUSSION THREAD

**Discussion by Megh-Shah-2901** [community/community#110482] [GitHub Education]
### Hi everyone,

I have been using this developer pack for around last 2-3 years but now I was asked to re-verify. I made a request for re-verification and it was approved on 27th February 2024. But, I have still not received the access to the platforms and solutions. How much time do it take between the request approval and getting access to the tools?

**Comment by ZuhairTarif** [comment]
approx 3 days, sometimes it take times

**Reply by Megh-Shah-2901** [reply]
Thanks for your help
